Automattic/newspack-blocks
# Update Calypso Build to 5.1.0

### All Submissions:






### Changes proposed in this Pull Request:

This PR updates `calypso-build` to the latest version, 5.1.0. 

The update removes a few Sass Muriel color variables that the Donate block uses. I didn't want to introduce any style changes in this PR so I created a temporary Sass file with these variables (https://github.com/Automattic/newspack-blocks/compare/update/calypso-build-5.1.0?expand=1#diff-d7cc79577ac5a9dd72abcdbc89faa614). In a future PR we should replace the Muriel color values and remove this file, cc: @thomasguillot. 

The approach to dependency extraction changes in this update to use https://developer.wordpress.org/block-editor/packages/packages-dependency-extraction-webpack-plugin/. 

### How to test the changes in this Pull Request:

1. `npm ci`. Verify `calypso-build` updates to version 5.1.0.
2. `npm run start`. Verify all blocks build with no errors.
3. Test normal operation of all three blocks in editor and view. Should be identical to `master`.
4. Bonus ⭐️: None of the blocks have view-side scripts. Add some Javascript to one  of  the `view.js` files and verify that it executes on the  front end. To verify the dependency enqueuing import a library. E.G.:

```
import classNames from 'classnames';
import { Component } from '@wordpress/element';
console.log( classNames( 'hello', 'classnames' ) );
```

### Other information:
